html/docs/files.html

```diff
@@ -95,6 +95,19 @@ <h4>Setup</h4>
               only affects your ability to <code>undo</code> changes.
             </p>
 
+						<p> 
```

Comment:
Please make the indentation consistent the rest of the file. Do not use tabs but spaces.
Also format your text to "one line per sentence". This is not how the other files are formatted but we will migrate towartd it and it makes version control of changes (which mostly apply to sentences) easier.